lib/safe-pg-migrations/plugins/old_rails_version_support.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SafePgMigrations
+  module OldRailsVersionSupport
+    ACTIVE_RECORD_VERSION = ActiveRecord::VERSION::STRING
+
+    ruby2_keywords def validate_foreign_key(from_table, to_table = nil, **options)
+      if ACTIVE_RECORD_VERSION < '6'
```

Comment:
You should not use a string for comparison, otherwise it will break starting from `'10'`

Reply:
Fixed :)